sdcm/ycsb_thread.py

```diff
@@ -264,9 +264,14 @@ def _run_stress(self, loader, loader_idx, cpu_idx):  # pylint: disable=too-many-
                                    extra_docker_opts=f'--label shell_marker={self.shell_marker}',
                                    docker_network=self.params.get('docker_network'))
                 dns_options += f'--dns {dns.internal_ip_address} --dns-option use-vc'
+            os.makedirs(f'/tmp/hdr-output-directory/{loader_idx}/{cpu_idx}', exist_ok=True)
```

Comment:
something doesn't add up here
why are we creating this directory like that ? the docker gonna run on the loader

it should be something like:
```python
tmp_hdr_dir  = loader.sudo(f"mktemp -d -t {loader_idx}_{cpu_idx} ").stdout
```